doc/api/cli.md

```diff
@@ -138,6 +138,10 @@ Force FIPS-compliant crypto on startup. (Cannot be disabled from script code.)
 
 Specify ICU data load path. (overrides `NODE_ICU_DATA`)
 
+### `--preserve-symlinks`
+
+Instructs the module loader to preserve symbolic links when resolving and
+caching modules.
```

Comment:
Can you add one or two lines explaining when it makes a difference (e.g. `npm link`)?